core/txbuilder/witness.go

```diff
@@ -128,9 +128,9 @@ func (sw *SignatureWitness) Sign(ctx context.Context, tpl *Template, index int,
 		if !contains(xpubs, keyID.XPub) {
 			continue
 		}
-		var path [][]byte
-		for _, p := range keyID.DerivationPath {
-			path = append(path, p)
+		path := make([]([]byte), len(keyID.DerivationPath))
```

Comment:
Why not `make([][]byte, len(keyID.DerivationPath))`?

It's just possible that `make([][]byte, 0, len(keyID.DerivationPath))` is a hair better in performance, because then the Go runtime doesn't have to initialize that space to all `nil`s. (You would then revert the subsequent lines to be the same as before the diff.)